core/templates/services/math-interactions.service.spec.ts

```diff
@@ -124,36 +124,36 @@ describe('MathInteractionsService', () => {
 
     expect(mathInteractionsService.validateExpression('(x+y)/0')).toBeFalse();
     expect(mathInteractionsService.getWarningText()).toBe(
-      'Division by zero not allowed.');
+      'Your answer includes a division by zero, which is not valid.');
 
     expect(mathInteractionsService.validateExpression(
       '(x+y)/(y-y)')).toBeFalse();
     expect(mathInteractionsService.getWarningText()).toBe(
-      'Division by zero not allowed.');
+      'Your answer includes a division by zero, which is not valid.');
 
     expect(mathInteractionsService.validateExpression('a)(b')).toBeFalse();
     expect(mathInteractionsService.getWarningText()).toBe(
       'It looks like your answer has an invalid bracket pairing.');
 
     expect(mathInteractionsService.validateExpression('a_2 + 3')).toBeFalse();
     expect(mathInteractionsService.getWarningText()).toBe(
-      'Invalid character \'_\' present in the expression.');
+      'Your answer contains an invalid character: "_".');
 
     expect(mathInteractionsService.validateExpression(
       '3.4.5 + 45/a')).toBeFalse();
     expect(mathInteractionsService.getWarningText()).toBe(
-      'Invalid integer.');
+      'Your answer contains an invalid integer: 3.4.5');
```

Comment:
invalid term you mean? 3.4.5 is not an integer

Reply:
Yup, makes sense. Done.